tcms/core/management/commands/upgrade.py

```diff
@@ -0,0 +1,86 @@
+from django.core.management import call_command
+from django.core.management.base import BaseCommand
+
+
+class Command(BaseCommand):
+    help = "Perform post-upgrade tasks for Kiwi TCMS"
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--noinput",
+            "--no-input",
+            "--yes",
+            "-y",
+            action="store_false",
+            dest="interactive",
+            help="Automatic mode. Does not require user confirmation",
+        )
+
+    def handle(self, *args, **kwargs):
+        output = None
+        if kwargs["verbosity"]:
+            output = self.stdout
+        answer = "y"
+        if kwargs["interactive"]:
+            answer = "x"
+
+        if output:
+            self.stdout.write(
+                """To finish the upgrade process, the following
+management commands will be executed:
+
+migrate
+refresh_permissions
+delete_stale_attachments
+delete_stale_comments
+remove_stale_contenttypes
+clean_orphan_obj_perms
```

Comment:
nitpick: 

Just for consistency sake I prefer to group the commands, e.g. refresh_permissions, then remove object perms and content types, etc.  The only exception should be if a previous command needs to be executed before a subsequent one.